src/packages/shared-types/actions.ts

```diff
@@ -10,6 +10,14 @@ export enum Action {
   WITHDRAW_PACKAGE = "withdraw-package",
 }
 
+export type ActionTypes =
+  | "enable-rai-withdraw"
+  | "withdraw-package"
+  | "disable-rai-withdraw"
+  | "issue-rai"
+  | "withdraw-rai"
+  | "respond-to-rai";
+
```

Comment:
Can we set ourselves up for a refactor of these enums into unions? I think this is something we can go-ahead with just to stop us from further using the enums in new files. 

If it's cool with y'all, I can write the ticket up and catalog the enums we use right now to get opinions on what stays and what can just be a union. 

Cc: @13bfrancis @pkim-gswell @mdial89f